Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+PROJECT = 'mail'
+
+pipeline {
+  agent any
+
+  options {
+    ansiColor('xterm')
+    disableConcurrentBuilds()
+    buildDiscarder(logRotator(numToKeepStr: '10')) // only keep artifacts of the last 10 builds
+  }
+
+  parameters {
+    booleanParam(name: 'PRE_NOTIFY', defaultValue: true, description: 'Notify of start of build for master branch')
+    booleanParam(name: 'CHECK_SEMAPHORE', defaultValue: true, description: 'Check current Semaphore tests status')
```

Comment:
Can remove this since we won't be checking Semaphore.

Reply:
Fixed